cheatsheets_draft/Multifactor_Authentication_Cheat_Sheet.md

```diff
@@ -0,0 +1,328 @@
+| Something You Are | Fingerprints, facial recognition, iris scans and handprint scans. |
+| Location | Source IP ranges and geolocation |
+
+It should be emphasised that while requiring multiple examples of a single factor (such as needing both a password and a PIN) **does not constitute MFA**, although it may provide some security benefits over a simple password.
+
+Additionally, while the following sections discuss the disadvantage and weaknesses of various different types of MFA, in many cases these are only relevant against targeted attacks. **Any MFA is better than no MFA**.
+
+# Advantages
+
+The most common way that user accounts get compromised on applications is through weak, re-used or stolen passwords. Despite any technical security controls implemented on the application, users are liable to choose weak passwords, or to use the same password on different applications. As developers or system administrators, it should be assumed that users' passwords will be compromised as some point, and the system should be designed in order to defend against this.
+
+Multi-factor authentication (MFA) is by far the best defense against the majority of password-related attacks, including brute-force, [credential stuffing](Credential_Stuffing_Prevention_Cheat_Sheet.md) and password spraying, with analysis by Microsoft suggesting that it would have stopped [99.9% of account compromises](https://techcommunity.microsoft.com/t5/Azure-Active-Directory-Identity/Your-Pa-word-doesn-t-matter/ba-p/731984).
+
+# Disadvantages
+
+The biggest disadvantage of MFA is the increase in management complexity for both administrators and end users. Many less technical users may find it difficult to configure and use MFA. Additionally, there are a number of other common issues encountered:
+
+- Types of MFA that require users to have specific hardware can introduce significant costs and administrative overheads.
+- Users may become locked out of their accounts if they lose or are unable to use their other factors.
+- MFA introduces additional complexity into the application.
+- Many MFA solutions add external dependencies to systems, which can introduce security vulnerabilities or single points of failure.
+- Processes implemented to allow users to bypass or reset MFA may be exploitable by attackers.
+- Requiring MFA may prevent some users from accessing the application.
+
+# Quick Recommendations
+
+Exactly when and how MFA is implemented in an application will vary on a number of different factors, including the threat model of the application, the technical level of the users, and the level of administrative control over the users. These need to be considered on a per-application basis.
+
+However, the following recommendations are generally appropriate for most applications, and provide an initial starting point to consider.
+
+- Provide the option for users to enable MFA on their accounts using [TOTP](#software-otp-tokens).
```

Comment:
broken link it should be `#software-totp-tokens`

Reply:
Fixed in e917e0b